packages/one-app-bundler/webpack/app/webpack.client.js

```diff
@@ -55,73 +57,78 @@ const getCoreJsModulePaths = (targets) => {
       .filter((moduleName) => moduleNames.includes(moduleName)).length);
 };
 
-module.exports = (babelEnv) => merge(
-  common,
-  {
-    output: {
-      path: path.resolve(packageRoot, `build/app/tmp${babelEnv !== 'modern' ? '/legacy' : ''}`),
-      filename: '[name].js',
-    },
-    entry: {
-      app: './src/client/client',
-      vendors: [
-        ...babelEnv !== 'modern' ? ['cross-fetch/polyfill', 'url-polyfill', 'abort-controller/polyfill'] : [],
-        ...(babelEnv !== 'modern' ? getCoreJsModulePaths(legacyBrowserList) : getCoreJsModulePaths(browserList)).map(resolve),
-        resolve('regenerator-runtime/runtime'),
-        ...Object.keys(moduleExternals).map(resolve),
-      ],
-    },
-    resolve: {
-      alias: {
-        'transit-js': path.resolve(packageRoot, 'src/universal/vendors/transit-amd-min.js'),
+module.exports = (babelEnv) => {
+  const configOptions = getConfigOptions();
+  const disableDevelopmentLegacyBundle = configOptions.disableDevelopmentLegacyBundle && process.env.NODE_ENV === 'development';
+
+  return merge(
+    common,
+    {
+      output: {
+        path: path.resolve(packageRoot, `build/app/tmp${babelEnv !== 'modern' && !disableDevelopmentLegacyBundle ? '/legacy' : ''}`),
```

Comment:
`babelEnv !== 'modern' && !disableDevelopmentLegacyBundle`
Can we put this in a var since its used twice in this function

Reply:
Also pulled `babelEnv !== 'modern'` into its own variable as well.